### ISSUE TITLE
Markers not stored anymore after Dynmap reload

### ISSUE THREAD

**Issue by stormboomer** [webbukkit/dynmap#2496]
## Server Information

- I can reproduce this issue consistently
     - [x] In Multiplayer 
     - [ ] After deleting/renaming the DYNMAP folder
- [x] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I'm having the same problem.
- [ ] I am crashing and can provide my crash report(s)
- I have tried the following diagnostics;
     - [ ] Re-rendered missing chunks using /dynmap fullrender
     - [ ] Cleared cached tiles using /dynmap purgeworld [world] (especially if using a map from an older version)
     - [ ] Deleted/Renamed the dynmap folder to regenerate configs and files fresh
     - [ ] Tested using the 'HEAD' version from dynmap.us/builds (these are stable to use)
     - [ ] Rechecked the Minecraft Server and Webserver Config (If using an external Webserver)

## Issue Description
When working mit Markers there seems to be a problem. New Markers are not stored in markers.yml after someone did `/dynmap reload`. After a clean server restart its possible to add markers again. It feels like there is a problem writing the markers to the markers.yml, sadly there is no error in the console.
Also when trying to add duplicate marker after `/dynmap reload` the marker duplicate is not created but there is an error message send to the client `Error creating marker`. This is an indicator for me that the markers are actually stored in RAM but not on filesystem. The Message also appears when adding dupplicate markers before doing `/dynmap reload`

## Reproduce Steps
1. Add marker via dmarker command
2. issue /dynmap reload
3. Add marker via dmarker command
4. check markers.yml only contains first marker

**Comment by crizyx** [comment]
Same problem since 1.12

**Comment by crizyx** [comment]
I have the same problem since 1.12

**Comment by stormboomer** [comment]
I think I have localized the Issue.
The problem is, that the Scheduled Task for saving the changes for markers is not restarted after a reload. I was able to Fix the issue with changing the following stuff:
in DynmapCore `org.dynmap.markers.impl.MarkerAPIImpl` changing scheduleWriteJob to public
```java
    public void scheduleWriteJob() {
        core.getServer().scheduleServerTask(new DoFileWrites(), 20);
    }
```
in DynmapCore `org.dynmap.DynmapCore` changing markerapi to public
```java
    public MarkerAPIImpl   markerapi;
```
for Spigot `org.dynmap.bukkit.DynmapPlugin` changing the OnEnable function. (adding check at the top + Call the start for Scheduled Write Job) probably needs to be done for all the other Server versions aswell.
```java
    public void onEnable() {
        if(core != null){
            if(core.markerapi != null){
                core.markerapi.scheduleWriteJob();
            }
        }
...
```

Because I'm a total noob with github and gradle I would like to ask one of the developer to include the changes. Also it might be better to use getter / starter methods to do this, but i was just interested in a fast solution.
